src/circuit/gadget/ecc/chip/add.rs

```diff
@@ -1,8 +1,7 @@
 use super::{util, CellValue, EccConfig, EccPoint};
 
-use group::Curve;
 use halo2::{
-    arithmetic::{CurveAffine, CurveExt, Field, FieldExt},
+    arithmetic::FieldExt,
     circuit::Region,
     plonk::{ConstraintSystem, Error, Expression},
 };
```

Comment:
On line 75, the label for the gate does not match the comment.

Reply:
Also the labels on lines 85 and 92 are a bit confusing; I suggest replacing those with the corresponding comments.

Reply:
Also I think the gate on line 116 may be incorrect.

Reply:
Resolving this comment since `add.rs` has undergone significant fixes.